### PR TITLE
feat: add file associations patch

### PR DESCRIPTION
**Pull request by rubiin**

### CODE REVIEW

File: Hyde [hunk 1]
```diff
@@ -532,6 +532,13 @@ inject() { #? User/Device specific scripts that might be useful
         shift
         SUPER /usr/lib/hyde-cli/Install-chaotic-aur "${@}"
         ;;
+    file_associations) #? Add file associations
+        shift
```
Comment (kRHYME7) [inline Hyde:536]: not needed

**Comment by kRHYME7** [comment]
BTW I fixed it in my end by 

` 
curl -L https://raw.githubusercontent.com/KDE/plasma-workspace/master/menu/desktop/plasma-applications.menu -o $HOME/.config/menus/applications.menu
`

So now I'm considering the above solution as it is lightweight.

**Comment by rubiin** [comment]
> BTW I fixed it in my end by
> 
> ` curl -L https://raw.githubusercontent.com/KDE/plasma-workspace/master/menu/desktop/plasma-applications.menu -o $HOME/.config/menus/applications.menu`
> 
> So now I'm considering the above solution as it is lightweight.
> 
> Can you confirm it?
` kbuildsycoca6 --noincremental` you have to run this i guess

**Comment by kRHYME7** [comment]
Which package have this? kbuild?

**Comment by rubiin** [comment]
its a kde service itself i guess

**Comment by kRHYME7** [comment]
So after this https://github.com/kRHYME7/Hyde-cli/pull/45#issuecomment-2089360027

run  ` kbuildsycoca6 --noincremental  ` ?

**Comment by rubiin** [comment]
yup. `kbuildsycoca6` builds binary cache of the data stored in .desktop and MIME type .xml files that the KService framework uses to find plugins, applications and other services.

**Comment by kRHYME7** [comment]
I see okk

**Comment by rubiin** [comment]
I included this on the cli here as the this will update a user mimelist with defaults when they accidently  delete the file. 
Although this can be achieved with a function on the user zshrc, putting this on cli seems like a better option so user wont complain with this sort of mishap in future

**Comment by rubiin** [comment]
closing in favor of https://github.com/prasanthrangan/hyprdots/commit/d80c54eb17f1b7f630ed64b03e8c15750a65c9ec#diff-bb0d99a7f842baa2da12a26ed7c32798fdfe426d6b48ef435cf3d0e3c1971039

**Pull request closed without merging**